data/json/recipes/recipe_others.json

```diff
@@ -3531,7 +3531,7 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [
       [ [ "salt_water", 2 ], [ "saline", 10 ], [ "salt", 10 ] ],
-      [ [ "raw_leather", 1 ], [ "raw_tainted_leather", 1 ], [ "raw_hleather", 1 ] ]
+      [ [ "raw_leather", 1 ], [ "raw_tainted_leather", 50 ], [ "raw_hleather", 50 ] ]
```

Comment:
Is this supposed to be `[ "raw_leather", 1 ]`?

Reply:
Nope. Thanks!